x-pack/plugins/index_lifecycle_management/public/application/sections/policy_table/components/policy_table/add_policy_to_template_confirm_modal.js

```diff
@@ -105,57 +65,41 @@ export class AddPolicyToTemplateConfirmModal extends Component {
         <EuiSpacer size="s" />
       </Fragment>
     );
-  }
-  getSelectedTemplate() {
-    const { templates, templateName } = this.state;
-    return find(templates, (template) => template.name === templateName);
-  }
-  renderForm() {
-    const { templates, templateName, templateError } = this.state;
-    const options = templates.map(({ name }) => {
-      return {
-        value: name,
-        text: name,
-      };
-    });
-    options.unshift({
-      value: '',
-      text: i18n.translate(
-        'xpack.indexLifecycleMgmt.policyTable.addLifecyclePolicyToTemplateConfirmModal.chooseTemplateMessage',
-        {
-          defaultMessage: 'Select an index template',
-        }
-      ),
-    });
+  };
+
+  const renderUnableToLoadTemplatesCallout = () => {
+    const { statusCode = '', message = '' } = error;
     return (
-      <EuiForm>
-        {this.renderTemplateHasPolicyWarning()}
-        <EuiFormRow
-          isInvalid={!!templateError}
-          error={templateError}
-          label={
+      <Fragment>
+        <EuiSpacer size="s" />
+        <EuiCallOut
+          style={{ maxWidth: 400 }}
+          title={
             <FormattedMessage
-              id="xpack.indexLifecycleMgmt.policyTable.addLifecyclePolicyToTemplateConfirmModal.chooseTemplateLabel"
-              defaultMessage="Index template"
+              id="xpack.indexLifecycleMgmt.policyTable.addLifecyclePolicyToTemplateConfirmModal.errorLoadingTemplatesTitle"
+              defaultMessage="Unable to load index templates"
             />
           }
+          color="danger"
         >
-          <EuiSelect
-            options={options}
-            value={templateName}
-            onChange={(e) => {
-              this.setState({ templateError: null, templateName: e.target.value });
-            }}
-          />
-        </EuiFormRow>
-        {this.renderAliasFormElement()}
-      </EuiForm>
+          <p>
+            {message} ({statusCode})
+          </p>
+          <EuiButton isLoading={isLoading} color="danger" onClick={sendRequest}>
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyToTemplateConfirmModal.errorLoadingTemplatesButton"
+              defaultMessage="Try again"
+            />
+          </EuiButton>
+        </EuiCallOut>
+        <EuiSpacer size="s" />
+      </Fragment>
     );
-  }
-  renderAliasFormElement = () => {
-    const { aliasName } = this.state;
-    const { policy } = this.props;
-    const showAliasTextInput = policy && get(policy, 'policy.phases.hot.actions.rollover');
+  };
+
+  const renderAliasFormElement = () => {
+    console.log(policy);
+    const showAliasTextInput = policy?.policy?.phases.hot?.actions.rollover;
```

Comment:
The original code defines this without the first few optional chains. Do we need them or can we match the original?

```
const showAliasTextInput = policy.policy.phases.hot?.actions.rollover;
```